dlt/load/load.py

```diff
@@ -196,7 +196,9 @@ def w_spool_job(
     def spool_new_jobs(self, load_id: str, schema: Schema) -> Tuple[int, List[LoadJob]]:
         # use thread based pool as jobs processing is mostly I/O and we do not want to pickle jobs
         load_files = filter_new_jobs(
-            self.load_storage.list_new_jobs(load_id), self.destination.capabilities(), self.config
+            self.load_storage.list_new_jobs(load_id),
+            self.destination.capabilities(self.initial_client_config),
```

Comment:
I had to pass config here to get the correct adjusted capabilities (see `qdrant/factory.py` changes). Please double check.

Reply:
I even improved it and requested full (not just initial) config here